packages/forms/src/Components/RichEditor/RichContentRenderer.php

```diff
@@ -236,6 +238,22 @@ protected function processMergeTags(Editor $editor): void
         });
     }
 
+    public function processNodesUsing(Closure $callback): static
+    {
+        $this->nodeProcessor = $callback;
```

Comment:
I think you should be able to call the method multiple times to create a chain of processors